programs/ssl/ssl_server2.c

```diff
@@ -957,6 +960,14 @@ int main( int argc, char *argv[] )
     char *p, *q;
     const int *list;
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( mbedtls_platform_setup( &platform_ctx ) != 0 )
+    {
+        mbedtls_printf( "Failed initializing platform.\n" );
```

Comment:
We should really display the return code here, similar to other test programs, and other places within this example.